pandas/core/frame.py

```diff
@@ -9836,6 +9840,8 @@ def applymap(
         --------
         DataFrame.apply : Apply a function along input axis of DataFrame.
         DataFrame.replace: Replace values given in `to_replace` with `value`.
+        DataFrame.applymap : Apply a function elementwise on a DataFrame (deprecated).
```

Comment:
I'm okay with this staying, but I think it can be removed.

Reply:
I don't have strong opinion either, I can remove if I need to update the PR?